documentation/rfc/ptref_grammar.md

```diff
@@ -173,7 +173,7 @@ The graph is as follow (edges may have weight to avoid unwanted behavior):
le%3D%22bold%22%5D%0A%20%20%20%20%20%20%20%20MetaVehicleJourney%20-%3E%20Impact%20%5Bstyle%3D%22bold%22%5D%0A%20%20%20%20%20%20%20%20Impact%20-%3E%20Network%20%20%5B%20style%3D%22dashed%22%20fontcolor%3D%22green4%22%20color%3D%22green4%22%20%5D%0A%20%20%20%20%20%20%20%20Impact%20-%3E%20Line%20%20%5B%20style%3D%22dashed%22%20fontcolor%3D%22green4%22%20color%3D%22green4%22%20%5D%0A%20%20%20%20%20%20%20%20Impact%20-%3E%20Route%20%20%5B%20style%3D%22dashed%22%20fontcolor%3D%22green4%22%20color%3D%22green4%22%20%5D%0A%20%20%20%20%20%20%20%20Impact%20-%3E%20StopArea%20%20%5B%20style%3D%22dashed%22%20fontcolor%3D%22green4%22%20color%3D%22green4%22%20%5D%0A%20%20%20%20%20%20%20%20Impact%20-%3E%20StopPoint%20%20%5B%20style%3D%22dashed%22%20fontcolor%3D%22green4%22%20color%3D%22green4%22%20%5D%0A%20%20%20%20%20%20%20%20Impact%20-%3E%20MetaVehicleJourney%20%20%5B%20style%3D%22dashed%22%20fontcolor%3D%22green4%22%20color%3D%22green4%22%20%5D%0A%20%20%20%20%7D%0A%0A%20%20%20%20subgraph%20cluster_1%20%7B%0A%20%20%20%20%20%20%20%20%23rank%3Dsink%0A%20%20%20%20%20%20%20%20%23rankdir%3DLR%0A%20%20%20%20%20%20%20%20node%20%5Bshape%3Dplaintext%5D%0A%20%20%20%20%20%20%20%20style%3D%22filled%22%0A%20%20%20%20%20%20%20%20style%3Dfilled%0A%09%09color%3Dlightgrey%0A%20%20%20%20%20%20%20%20%23label%20%3D%20%22Legend%22%3B%0A%20%20%20%20%20%20%20%20key%20%5Blabel%3D%3C%3Ctable%20border%3D%220%22%20cellpadding%3D%222%22%20cellspacing%3D%220%22%20cellborder%3D%220%22%3E%0A%20%20%20%20%20%20%20%20%20%20%3Ctr%3E%3Ctd%20align%3D%22right%22%20port%3D%22i1%22%3E%3Cfont%20color%3D%22black%22%3EDefault%20link%3C%2Ffont%3E%3C%2Ftd%3E%3C%2Ftr%3E%0A%20%20%20%20%20%20%20%20%20%20%3Ctr%3E%3Ctd%20align%3D%22right%22%20port%3D%22i2%22%3E%3Cfont%20color%3D%22red%22%3EShortcut%3C%2Ffont%3E%3C%2Ftd%3E%3C%2Ftr%3E%0A%20%20%20%20%20%20%20%20%20%20%3Ctr%3E%3Ctd%20align%3D%22right%22%20port%3D%22i3%22%3E%3Cfont%20color%3D%22green4%22%3EImpact%20link%3C%2Ffont%3E%3C%2Ftd%3E%3C%2Ftr%3E%0A%20%20%20%20%20%20%20%20%20%20%3C%2Ftable%3E%3E%5D%0A%20%20%20%20%20%20%20%20key2%20%5Blabel%3D%3C%3Ctable%20border%3D%220%22%20cellpadding%3D%222%22%20cellspacing%3D%220%22%20cellborder%3D%220%22%3E%0A%20%20%20%20%20%20%20%20%20%20%3Ctr%3E%3Ctd%20port%3D%22i1%22%3E%26nbsp%3B%3C%2Ftd%3E%3C%2Ftr%3E%0A%20%20%20%20%20%20%20%20%20%20%3Ctr%3E%3Ctd%20port%3D%22i2%22%3E%26nbsp%3B%3C%2Ftd%3E%3C%2Ftr%3E%0A%20%20%20%20%20%20%20%20%20%20%3Ctr%3E%3Ctd%20port%3D%22i3%22%3E%26nbsp%3B%3C%2Ftd%3E%3C%2Ftr%3E%0A%20%20%20%20%20%20%20%20%20%20%3C%2Ftable%3E%3E%5D%0A%20%20%20%20%20%20%20%20key%3Ai1%3Ae%20-%3E%20key2%3Ai1%3Aw%20%5B%20label%3D%22weight%3D1%22%20style%3D%22bold%22%5D%0A%20%20%20%20%20%20%20%20key%3Ai2%3Ae%20-%3E%20key2%3Ai2%3Aw%20%5B%20label%3D%22weight%3Dx%22%20fontcolor%3D%22red%22%20color%3D%22red%22%20%5D%0A%20%20%20%20%20%20%20%20key%3Ai3%3Ae%20-%3E%20key2%3Ai3%3Aw%20%5B%20label%3D%22weight%3D100%22%20style%3D%22dashed%22%20fontcolor%3D%22green4%22%20color%3D%22green4%22%20%5D%0A%20%20%20%20%20%20%7D%0A%20%20%0A%7D)
```

Comment:
Links with a weight of `1` are all displayed in black. Red is for links with a specific edge weight.

Reply:
Ok done